app/code/Magento/SalesRule/Model/Rule/Action/Discount/CartFixed.php

```diff
@@ -52,10 +47,9 @@ public function __construct(
         Validator $validator,
         DataFactory $discountDataFactory,
         PriceCurrencyInterface $priceCurrency,
-        DeltaPriceRound $deltaPriceRound,
+        private readonly DeltaPriceRound $deltaPriceRound,
         ?CartFixedDiscount $cartFixedDiscount = null
```

Comment:
Why not promote this property?
```suggestion
        protected CartFixedDiscount $cartFixedDiscountHelper = null,
```